v1.7/ics/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: IBM
+name: IBM Container Service
```

Comment:
I believe this should now be `IBM Cloud Container Service`.  Given this, should we rename the directory to iccs?